kubernetes-client/src/main/java/io/fabric8/kubernetes/client/KubernetesClient.java

```diff
@@ -140,78 +123,6 @@
    */
   <T extends CustomResource, L extends KubernetesResourceList<T>> MixedOperation<T, L, Resource<T>> customResources(Class<T> resourceType, Class<L> listClass);
 
-
-  /**
-   * Typed API for managing CustomResources. You would need to provide POJOs for
-   * CustomResource into this and with it you would be able to instantiate a client
-   * specific to CustomResource.
-   *
-   * <p>
-   *   Note: your CustomResource POJO (T in this context) must implement
-   *   <a href="https://github.com/fabric8io/kubernetes-client/blob/master/kubernetes-model-generator/kubernetes-model-core/src/main/java/io/fabric8/kubernetes/api/model/Namespaced.java">
-   *     io.fabric8.kubernetes.api.model.Namespaced
-   *   </a> if it is a Namespaced scoped resource.
-   * </p>
-   *
-   * @param crdContext CustomResourceDefinitionContext describes the core fields used to search for CustomResources
-   * @param resourceType Class for CustomResource
-   * @param listClass Class for list object for CustomResource
-   * @param <T> T type represents CustomResource type. If it's Namespaced resource, it must implement
-   *           io.fabric8.kubernetes.api.model.Namespaced
-   * @param <L> L type represents CustomResourceList type
-   * @return returns a MixedOperation object with which you can do basic CustomResource operations
-   */
-  <T extends HasMetadata, L extends KubernetesResourceList<T>> MixedOperation<T, L, Resource<T>> customResources(CustomResourceDefinitionContext crdContext, Class<T> resourceType, Class<L> listClass);
```

Comment:
I don't see any deprecation annotation or JavaDoc tag for this method. Why did you remove it?

If it's not an error, the procedure would be to annotate it now with the `@Depreacted` annotation and remove it in future releases.

Reply:
These methods don't work as we need to specify annotations in POJOs https://github.com/fabric8io/kubernetes-client/issues/2738#issuecomment-763869356 . I don't think it's a good idea to keep methods that don't work at all.

Reply:
Fair enough, but that's out of scope of the current issue>PR.

Ideally you should create a separate PR for this, or at least put it in a different commit.

Reply:
okay, I'll remove this from this commit and create a separate PR in next update